tsconfig.json

```diff
@@ -2,12 +2,6 @@
   "extends": "./tsconfig.build.json",
 
   "compilerOptions": {
-    "baseUrl": "./packages",
-    "paths": {
-      // "@weknow-network/wk-lib-consumer-playground": ["wk-lib-consumer-playground"],
-      // "@weknow-network/wk-lib-consumer-playground-npm": ["wk-lib-consumer-playground-npm"],
-      "@weknow-network/wk-lib-playground": ["wk-lib-playground/src"],
-      "@weknow-network/*": ["*/src"]
-    }
+    "baseUrl": "./packages"
```

Comment:
I do not believe paths are not necessary to define in tsconfig here.  However if you are looking to consume your components directly from your library workspace ( rather than the `dist` that contains the compiled library), this might be a way to approach it.